src/utils/calculateMemorisedWordCount.ts

```diff
@@ -3,7 +3,8 @@ import type { MetWords } from "../types";
 function calculateMemorisedWordCount(metWords: MetWords) {
   const yourMemorisedWordCount =
     Math.round(
-      Object.values(metWords).filter((timesSeen) => timesSeen > 29).length
+      // the 2 below should be restored to 29, 2 is to help debugging
+      Object.values(metWords).filter((timesSeen) => timesSeen > 2).length
```

Comment:
As mentioned elsewhere, this needs to change, I'm just adding a comment here to help me keep track of it.